elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchSort.java

```diff
@@ -62,6 +62,8 @@
     for (RelFieldCollation fieldCollation : collation.getFieldCollations()) {
       final String name = fields.get(fieldCollation.getFieldIndex()).getName();
       final String rawName = implementor.expressionItemMap.getOrDefault(name, name);
+      // if nulls order is not specified, default NULLS LAST/FIRST for ASC/DESC
+      implementor.addNullsSort(rawName, fieldCollation.nullDirection);
```

Comment:
The default behavior is stay same as ES ? Assording [_missing_values](https://www.elastic.co/guide/en/elasticsearch/reference/current/sort-search-results.html#_missing_values) The default behavior is always _LAST.

Reply:
No, it's not the same.
I think it would be better to follow SQL behavior of NULLS order(NULLS LAST/FIRST for ASC/DESC), since Elasticsearch Adapter converts SQL semantics to ES scripts.

Reply:
Ok. I get it.